mystbin/backend/mystbin_models.py

```diff
@@ -0,0 +1,43 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING, List, Optional, TypedDict
+
+from fastapi import Request
+from starlette.datastructures import State
+
+from utils.db import Database
+
+
+if TYPE_CHECKING:
+    from app import MystbinApp
+
+__all__ = (
+    "MystbinState",
+    "MystbinRequest",
+)
+
+
+class _StateUser(TypedDict):
+    id: int
+    token: Optional[str]
+    emails: Optional[List[str]]
```

Comment:
```suggestion
    emails: List[str]
```

Reply:
This one I was unsure of, does a `TEXT[]` column in psql default to an empty array, or NULL?